Basic Select/Weather-Observation-Station-10.sql

```diff
@@ -1,8 +1,9 @@
-# Author: Thomas George Thomas
+# Author: Ishan Deshpande
```

Comment:
Please could you move it to just above your solution. Also comments in SQL start with "-". Please update this line to use it. Thanks